### PR TITLE
Swift3

### PR DESCRIPTION
**Pull request by billyto**
Moving this pod to Swift 3, once the CR is OK, we will ask to be merged with the public repo.

### CODE REVIEW

**Comment by damienvieira** [comment]
LGTM

File: SwiftyJSONAPITests/JSONAPIDocumentTests.swift [hunk 1]
```diff
@@ -67,15 +67,15 @@ class JSONAPIDocumentTests: XCTestCase {
         let meta = document.meta!
         let keys = [String](meta.keys)
     
-        XCTAssertNotNil(meta, "document should have meta information")
-        XCTAssertEqual(keys[0], "authors", "meta should contain a authors key")
+        XCTAssertNotNil(meta, "document should have meta information")        
+        XCTAssertTrue(keys.contains("authors"),"meta should contain a authors key")
```
Comment (macohn) [inline SwiftyJSONAPITests/JSONAPIDocumentTests.swift]: 👍

**Comment by macohn** [comment]
LGTM

File: SwiftyJSONAPI/JSONAPIError.swift [hunk 2]
```diff
@@ -9,32 +9,32 @@
 import Foundation
 
 
-public class JSONAPIError: JSONPrinter {
+open class JSONAPIError: JSONPrinter {
```
Comment (damienvieira) [inline SwiftyJSONAPI/JSONAPIError.swift]: I think you are right, in some cases it probably doesn't make sense to make a class or property open, but instead should remain public. Any reason why we are making everything open @billyto ?

Comment (billyto) [inline SwiftyJSONAPI/JSONAPIError.swift]: "Public" to "Open" was done by the conversion tool. In this case, I think is fine to let future generations to run free and subclass any class. Unless we want to start rethinking the architecture of the project, which is not the intention of this PR.

**Pull request merged**